packages/integration-tests/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -310,7 +307,12 @@ async function setupScenario(kind: string, input: Input, scenario: Scenario) {
     ${datasourceBlock}
   `
 
-  const introspectionResult = await engine.introspect(schemaBase)
+  const engine = new MigrateEngine({
+    projectDir: process.cwd(),
+  })
+  const introspectionResult = await engine.introspect({ schema: schemaBase })
+  engine.stop()
```

Comment:
Is this really necessary here? Doesn't `introspect()` take care of that itself?

Reply:
I think @Jolg42 ninjed you, I have fixed this already while you were writing this :)

Reply:
I can live with that.